streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java

```diff
@@ -30,7 +30,7 @@
  * signals the processing pipeline to continue processing more records.
  */
 public class LogAndContinueExceptionHandler implements DeserializationExceptionHandler {
-    private static final Logger log = LoggerFactory.getLogger(StreamThread.class);
+    private static final Logger log = LoggerFactory.getLogger(LogAndContinueExceptionHandler.class);
```

Comment:
Fix checkstyle error due to unused StreamThread import

Reply:
I fix that，but another error appear. If there is any exception in test of testNoConsumeWithoutDescribeAclViaSubscribe, i have only change the class name of logger and do nothing for zookeeper.set.acl

Reply:
that may be transient error.. we can run the tests again by commenting "retest this please"

Reply:
thanks

Reply:
Execuse me,is there any transient error in the test. I can not see the result now